### PR TITLE
Nvidia container runtime

### PR DESCRIPTION
**Pull request by frezbo**
Add required components for nvidia container toolkit

Signed-off-by: Noel Georgi <git@frezbo.dev>

### CODE REVIEW

File: nvidia-container-toolkit/container-cli/patches/libnvidia-container/nvc_ldcache.c.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git src/nvc_ldcache.c src/nvc_ldcache.c
+index d73d0f1..c28e982 100644
+--- src/nvc_ldcache.c
++++ src/nvc_ldcache.c
+@@ -349,7 +349,7 @@ nvc_ldcache_update(struct nvc_context *ctx, const struct nvc_container *cnt)
+         if (validate_args(ctx, cnt != NULL) < 0)
+                 return (-1);
+ 
+-        argv = (char * []){cnt->cfg.ldconfig, cnt->cfg.libs_dir, cnt->cfg.libs32_dir, NULL};
++        argv = (char * []){cnt->cfg.ldconfig, cnt->cfg.libs_dir, cnt->cfg.libs32_dir, "-C", "/etc/ld.so.cache", NULL};
```
Comment (frezbo) [inline nvidia-container-toolkit/container-cli/patches/libnvidia-container/nvc_ldcache.c.patch:10]: https://gitlab.com/nvidia/container-toolkit/libnvidia-container/-/issues/1 tracked here

**Review by smira — APPROVED** [review]
:tada: awesome work!

**Review by andrewrynhard — APPROVED** [review]
Wow this is a lot of work. Great job!

**Pull request merged**